includes/definitions/lenovoXClarityIMPI.yaml

```diff
@@ -0,0 +1,21 @@
+os: lenovoXClarityIMPI
```

Comment:
I don't think introducing camel case OS names into our existing kebab case os names is a good idea.

Please rename the OS to `lenovo-xclarity` or `xclarity`  Thanks.